packages/notifications/examples/basic.md

```diff
@@ -0,0 +1,240 @@
+            >
+              <Label isRegular>Info</Label>
+            </Radio>
+          </Field>
+        </div>
+      </Well>
+    </Col>
+    <Col>
+      <Alert type={state.type}>
+        <Title>{capitalize(state.type)} Alert</Title>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et dolore magna.
+        <Close onClick={() => alert('closing alert')} aria-label="Close Alert" />
+      </Alert>
+    </Col>
+  </Row>
+</Grid>;
+```
+
+### Notification
+
+The `<Paragraph>` component should be used to wrap multi-line content within a `<Notification>`.
+Otherwise, no wrapper is necessary. Usage of `<Close />` must contain an `aria-label` or other
+assistive technique to have discernible text.
+
+```jsx
+const { Field, Label, Radio, Toggle } = require('@zendeskgarden/react-forms/src');
+
+const { MD } = require('@zendeskgarden/react-typography/src');
+
+const capitalize = string => string.charAt(0).toUpperCase() + string.slice(1);
+
+const initialState = {
+  type: 'success',
+  isMultiLine: true
+};
+
+<Grid>
+  <Row>
+    <Col size={12}>
+      <Well isRecessed>
+        <div role="group" aria-label="alert types radio">
+          <MD style={{ color: '#2f3941', fontWeight: 600 }}>Notification Types</MD>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="success"
+              checked={state.type === 'success'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Success</Label>
+            </Radio>
+          </Field>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="error"
+              checked={state.type === 'error'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Error</Label>
+            </Radio>
+          </Field>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="warning"
+              checked={state.type === 'warning'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Warning</Label>
+            </Radio>
+          </Field>
+          <Field>
+            <Radio
+              name="notification-type"
+              value="info"
+              checked={state.type === 'info'}
+              onChange={event => setState({ type: event.target.value })}
+            >
+              <Label isRegular>Info</Label>
+            </Radio>
+          </Field>
+        </div>
+      </Well>
+    </Col>
+    <Col>
+      <Well isRecessed>
+        <Field>
+          <Toggle
+            checked={state.isMultiLine}
+            onChange={event => setState({ isMultiLine: event.target.checked })}
+          >
+            <Label>Multi-line</Label>
+          </Toggle>
+        </Field>
+      </Well>
```

Comment:
Why is this in its own well rather than being combined with the controls above?

Reply:
It does look strange, I think since this wasn't associated to a prop, I left it on a separate `<Well>`. I moved it all into a single `<Well>`.